onnx/defs/gen_doc.py

```diff
@@ -17,18 +17,38 @@ def display_number(v):
     return str(v)
 
 
+def support_level_str(level):
+    return \
+        "[experimental] " if level == OpSchema.SupportType.EXPERIMENTAL else ""
+
+
 def main(args):
     args.output.write('## Operator Schemas\n')
-    args.output.write('*This file is automatically generated from the [def files](/onnx/defs)*\n')
+    args.output.write(
+        """*This file is automatically generated from the
+            [def files](/onnx/defs) via [this script](/onnx/defs/gen_doc.py).
+            Do not modify directly and instead edit operator definitions.*\n""")
+
+    # If support level is experimental, then don't generate documentation.
```

Comment:
Is this still true?

Reply:
Good catch, not true anymore after this change :-)